app/src/main/res/values/strings.xml

```diff
@@ -5,4 +5,6 @@
     <string name="back4app_server_url">https://parseapi.back4app.com/</string>
     <string name="back4app_app_id">DZcZ5t4E5OwIQLRyPNYvprBQyD1G0Oy5nQlqrtpz</string>
     <string name="back4app_client_key">rjp5DDCIPgdeIkkh5cbW5WlTD2SxObooqcDh5FZQ</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
is this being used for anything? if not remove in this PR

Reply:
hm weird I actually remember removing that line in the first PR, maybe it was added in due to an import or something? i'll remove it now though